README.md

```diff
@@ -1,14 +1,23 @@
 # QuickSubmit Plugin
 
-- OJS QuickSubmit Plugin
-- Version: (see version.xml)
+[![Build Status](https://travis-ci.com/pkp/quickSubmit.svg?branch=stable-3_4_0)](https://travis-ci.com/pkp/quickSubmit)
+[![OJS compatibility](https://img.shields.io/badge/ojs-3.5-brightgreen)](https://github.com/pkp/ojs/tree/stable-3_5_0)
```

Comment:
I think here should be stable-3_4_0 ?

Reply:
Yep! I've cherry-picked from main and didn't update this line and the one below, please update for me, I don't have write permissions here :)